crowbar_framework/app/models/neutron_service.rb

```diff
@@ -157,6 +159,34 @@ def validate_proposal_after_save proposal
       validation_error("The default tenant network type driver \"#{ml2_type_drivers_default_tenant_network}\" is not a selected ml2 type driver")
     end
 
+    if ml2_type_drivers.include? 'vxlan'
```

Comment:
This method is getting long; it would be nice to split it out into a separate method if possible.